packages/orchestration/test/examples/sendAnywhere.test.ts

```diff
@@ -101,7 +101,9 @@ test('send using arbitrary chain info', async t => {
       ...txChannelDefaults,
     },
   } as IBCConnectionInfo;
-  const chainName = await E(sendKit.creatorFacet).addChain(
+  const chainName = 'hot';
+  await E(sendKit.creatorFacet).addChain(
+    chainName,
```

Comment:
why?

Reply:
Oh I misread this as just pulling out the literal `'hot'` to name it `chainName` before passing it in. I see now that previously it was returned by the function.

👍 good change